### ISSUE TITLE
Can't validate on multiple tabs

### ISSUE THREAD

**Issue by bhavesh-vyas** [jonsamwell/angular-auto-validate#146]
Is there any way to validate form fields on non active tabs

**Comment by jonsamwell** [comment]
Yeah I think if you can get the form element you can validate it.

`validationManager.validateForm(formElement)`

**Comment by jonsamwell** [comment]
You would have to do something like the ngSubmit decorator

https://github.com/jonsamwell/angular-auto-validate/blob/master/src/config/ngSubmitDecorator.js#L25

**Comment by bhavesh-vyas** [comment]
Can you please share any link to working example using above function

Also, do i need different forms for different tabs? Right now I am using single form, and ng-show for the div's (inside the same form) which act as tab, like below

<form ng-submit="submit()">
  <div ng-show="current_tab=='tab_1'">.....</div>
  <div ng-show="current_tab=='tab_2'">.....</div>
  ...
</form>
